libs/jsonforms-components/src/lib/Context/index.tsx

```diff
@@ -89,12 +89,18 @@ interface FileManagement {
   downloadFile?: (file: File) => void;
   deleteFile?: (file: File) => void;
 }
+interface SubmitManagement {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  submitForm?: (any: any) => void;
```

Comment:
are the types really any?  There are way too many places in our code where "any" can be easily replaced with an actual type.  This makes the code much, much more managable.